Scripts/should-run-smoke-test.sh

```
@@ -0,0 +1,60 @@
+#/usr/bin/env bash
```

Shellcheck (0.11.0)

[error] Use #!, not just #, for the shebang.

(SC1113)